core/dbio/database/database.go

```diff
@@ -285,6 +286,8 @@ func NewConnContext(ctx context.Context, URL string, props ...string) (Connectio
 	} else if strings.HasPrefix(URL, "oracle:") {
 		conn = &OracleConn{URL: URL}
 		// concurrency = 2
+	} else if strings.HasPrefix(URL, "firebird:") || strings.HasPrefix(URL, "firebirdsql:") {
```

Comment:
- Why the duplicated code here? (see line 299).
- Also: IMHO just `firebird://` is enough. Any reason to support `firebirdsql://` url schema?